test/built-ins/Array/prototype/concat/is-concat-spreadable-proxy-revoked.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-isconcatspreadable
+es6id: 22.1.3.1.1
```

Comment:
Following that pattern would make the IDs largely redundant with the information encoded in the test file names.